spec/filtering.ts

```diff
@@ -4,6 +4,6 @@ import { field } from "../src/api/core/filteringApi";
 export function getRandomFilteringCriteria() {
   return faker.helpers.randomize([
     field("id").isEqualTo("1"),
-    (field: any) => field("someField").isDifferentFrom("someValue"),
+    (col: any) => col("someField").isDifferentFrom("someValue"),
```

Comment:
please enforce typing on `col`

Reply:
pointed type a bit better, but I couldn't enforce the whole type. Anyway, it is just a test helper :)